packages/react/src/components/pickers/Suggestions/Suggestions.types.ts

```diff
@@ -223,6 +224,12 @@ export interface ISuggestionsProps<T> extends React.Props<any> {
 
   /** Theme provided by High-Order Component. */
   theme?: ITheme;
+
+  /**
+   *  Refers to the removeButtonIconProps for the remove item action
+   *  @defaultvalue {iconName:'Cancel'}
```

Comment:
You'll need backticks here too
```suggestion
   *  Props for the icon used in the item's remove button.
   *  @defaultvalue `{ iconName:'Cancel' }`
```